clang/docs/ReleaseNotes.rst

```diff
@@ -399,6 +399,9 @@ Moved checkers
 Sanitizers
 ----------
 
+- ``-fsanitize=signed-integer-overflow`` now instruments signed arithmetic even
+  when ``-fwrapv`` is enabled. Previously, only division checks were enabled.
```

Comment:
Maybe suggest to add `-fno-sanitize=signed-integer-overflow` with `-fwrapv` if we users does not care about errors.

Reply:
Doesn't `-fno-sanitizer-signed-integer-overflow` already work? I did some [testing in godbolt](https://godbolt.org/z/oWdxvzs4P) and I am able to toggle this sanitizer on/off.

Reply:
It should work, it's about release note
As a hint if users who had `-fsanitize=undefined -fwrapv` and not they suddenly see reports.

Reply:
Gotcha, resolved with [0182698](https://github.com/llvm/llvm-project/pull/82432/commits/018269881647673b530b8cb4611c7a380a5a1b5c). Let me know if this is detailed enough 😄